at_commons/lib/src/verb/notify_fetch_verb_builder.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:at_commons/src/verb/verb_builder.dart';
+
+/// NotifyFetchVerbBuilder generated the command to fetches the notification
+/// with the given notification-id
+///
+/// ```
+/// e.g. To fetch the notification with the notification-id - '21d617cd-fb8a-43e4-98df-7e1536818d59'
+///
+/// var notifyFetchVerbBuilder = NotifyFetchVerbBuilder()
+///                                       ..notificationId = '21d617cd-fb8a-43e4-98df-7e1536818d59';
+/// ```
+class NotifyFetchVerbBuilder implements VerbBuilder {
+  /// Notification Id to fetch the notification
+  String? notificationId;
```

Comment:
Shouldn't be nullable. Can we require in constructor?

Reply:
Marked the notificationId as late.